### DISCUSSION TITLE
Welcome to cucumber7-ts-starter Discussions!

### DISCUSSION THREAD

**Discussion by hdorgeval** [hdorgeval/cucumber7-ts-starter#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ddanilovic5** [comment]
Hey guys,

I'm getting an error, always when I'm trying to debug some steps. The error is -
`Error: Parse error in 'tests/steps/{apiname}/testname.ts': (1:1): expected: #EOF, #Language, #TagLine, #FeatureLine, #Comment, #Empty, got 'import { Given, Then, When } from "@cucumber/cucumber";'
    at PassThrough.<anonymous> (/**/api-tests/node_modules/@cucumber/cucumber/src/cli/helpers.ts:73:11)`

Not sure way cucumber expects this file to be .feature file.
Any ideas what can cause this issue?
